distribution/kernel/pom.xml

```diff
@@ -165,6 +170,12 @@
                                     <destFileName>ddf-branding.jar</destFileName>
                                 </artifactItem>
                                 <artifactItem>
+                                    <groupId>org.ops4j.pax.logging</groupId>
+                                    <artifactId>pax-logging-log4j2</artifactId>
+                                    <version>${pax.logging.version}</version>
+                                    <outputDirectory>${setup.folder}/system/org/ops4j/pax/logging/pax-logging-log4j2/${pax.logging.version}</outputDirectory>
```

Comment:
The `karaf-maven-plugin` does not currently support setting `startLevel` for bundles in `startup.properties`, so that approach isn't appropriate at this time.